Frameworks/UIKit/UIWebView.mm

```diff
@@ -243,11 +243,31 @@ - (void)setDetectsPhoneNumbers:(BOOL)detect {
 }
 
 /**
- @Status Interoperable
+ @Status Gap
 */
 - (NSString*)stringByEvaluatingJavaScriptFromString:(NSString*)string {
-    NSString* ret = [_xamlWebControl invokeScript:@"eval" arguments:[NSArray arrayWithObject:string]];
-    return ret;
+    UNIMPLEMENTED_WITH_MSG(
+        "stringByEvaluatingJavaScriptFromString is not supported on our platform. Call evaluateJavaScript:completionHandler: instead.");
+    return StubReturn();
```

Comment:
Sure, I will add. Sorry, I missed this. Why the build did not break in Debug mode?